src/main/java/com/google/sps/servlets/ActionsServlet.java

```diff
@@ -0,0 +1,107 @@
+package com.google.sps.servlets;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.UnsupportedEncodingException;
+import java.nio.charset.StandardCharsets;
+import java.net.URLEncoder;
+import java.net.URL;
+import java.net.HttpURLConnection;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.HashMap;
+
+/** Servlet that actions items.*/
+@WebServlet("/actions")
+public class ActionsServlet extends HttpServlet {
+
+  private static final List<String> terms = Arrays.asList("Black Lives Matter", "COVID-19");
+  private static final String API_KEY = "API_KEY";  // Insert the API_KEY here for testing.
+  private static final String API_PATH = "https://api.globalgiving.org/api/public/services/search/projects/summary";
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    List<String> jsonResultList = new ArrayList<>();
+    for (String term : terms) {
+      String queryTerm = encodeTerm(term);
+      Map<String, String> queryParameters = new HashMap<>();
+      queryParameters.put("api_key", API_KEY);
+      queryParameters.put("q", queryTerm);
+      String jsonResult = urlQuery(API_PATH, queryParameters);
+      jsonResultList.add(jsonResult);
+    }
+    String jsonResultString = String.format("{\"results\": [%s]}", String.join(",", jsonResultList));
```

Comment:
[Nit] I mentioned this in PR #4 but there is a lot of duplicated code between different classes in terms of building JSON.  I think we should write a helper library to construct JSON, or, better yet, use an existing library such as https://docs.oracle.com/javaee/7/api/javax/json/JsonObjectBuilder.html.

This doesn't have to be done in this PR, but at the very least I would add a TODO to clean this up in the future.

Reply:
Currently I am developing the Actions classes. Once I have them I'm planning on using GSON to convert the objects to JSON. So, I think the TODO is not necessary as this was only the prototype.

Reply:
It's still useful to added TODOs to indicate how this will work in the future.  This gives better context to reviewers.  Maybe something like TODO(uribejaun): Factor this out into an Action class that uses GSON.